### ISSUE TITLE
[Idea]Mid only option

### ISSUE THREAD

**Issue by SunbleachedAngel** [ash47/LegendsOfDota#110]
I think it will be cool to add mid only, but not as gamemode but as an option in settings. And since mod has easy mod option, you can think off something with it.

**Comment by Veroalla** [comment]
Yeah, like enabling only mid lane creeps and making sidelane towers
immortal or untouchable

**Comment by ohos** [comment]
as additional option - disable neutral creeps

**Comment by ash47** [comment]
Will consider adding this at some point. I imagine it's just as easy as setting the top / bot towers to be invulnerable and removing the creep spawners.
